apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -122,28 +124,21 @@ public RifLoader(MetricRegistry appMetrics, LoadAppOptions options) {
    * @param metrics the {@link MetricRegistry} to use
    * @return a {@link HikariDataSource} for the BFD database
    */
-  static HikariDataSource createDataSource(LoadAppOptions options, MetricRegistry metrics) {
-    HikariDataSource dataSource = new HikariDataSource();
+  static HikariDataSource createDataSource(
+      DataSource dataSource, int maxPoolSize, MetricRegistry metrics) {
+    HikariDataSource hikariDataSource = new HikariDataSource();
 
     /*
-     * FIXME The pool size needs to be double the number of loader threads
-     * when idempotent loads are being used. Apparently, the queries need a
-     * separate Connection?
+     * FIXME The pool size needs to be double the number of loader threads when
+     * idempotent loads are being used. Apparently, the queries need a separate
+     * Connection?
```

Comment:
This comment would now be better placed at the point of call to `createDataSource()` since the parameter name here is literally `maxPoolSize` so the code here is semantically correct.

Reply:
Thanks! Done.